### DISCUSSION TITLE
Migrating from Sqlite to MariaDB error

### DISCUSSION THREAD

**Discussion by rickserraglia** [dani-garcia/vaultwarden#2363] [Q&A] [answered]
I followed the intire tutorial (Using the MariaDB Backend - https://github.com/dani-garcia/vaultwarden/wiki/Using-the-MariaDB-%28MySQL%29-Backend) and got this error when run vaultwarden:

"`DATABASE_URL` is a MySQL URL, but the 'mysql' feature is not enabled."

Could anyone help me? Please

**Comment by Masgalor** [comment, marked as answer]
The binary you are using was built without mysql support.

Please use the issue template to give us more information about your setup (deployment environment).

**Comment by BlackDex** [comment]
Looks like you either build Vaultwarden your self, or using a pre-build one which only used sqlite. The containers have all db endpoints enabled.

**Comment by rickserraglia** [comment]
I had installed by a script, and when I opened the code of the bash script I saw that it was only with sqlite and I modified to add mysql.
Sorry the delay to see the answers, is that had already solved the problem and did not come back to see the solution, but anyway, very thanks to all
